src/lib.rs

```diff
@@ -40,6 +40,7 @@ use std::cmp::max;
 #[cfg(feature = "abi-7-13")]
 use std::cmp::min;
 
+pub mod async_api;
```

Comment:
I'm not sure, but I think you can add `#[cfg(feature = "async_impl")]` here, and then you won't need to add it all over the `async_api` module

Reply:
i'll add an async_api flag, but i don't think it'll avoid that since building agnostic crates requires a separation between exposed api vs particular implementation i believe

Reply:
Ah, I see. I misunderstood why the `cfg`s were there. Either way works for me, so feel free to change it back in a future PR if you want